ui/app/AppLayouts/Profile/views/MyProfileView.qml

```diff
@@ -19,161 +21,69 @@ import StatusQ.Controls 0.1
 SettingsContentBase {
```

Comment:
This view doesn't behave well on the smallest possible viewport (@iurimatias @anastasiyaig @elina2015 I would add it as a test case):
![image](https://user-images.githubusercontent.com/33099791/176867653-c9bea3dc-6bd7-4a06-b5b0-f4cd3e77f385.png)

The quickest solution would be to change the minimumWidth of the application window, following @John-44 restrictions:
> The 'maximum' minimum window size we should support is 1280px e.g. so if it's useful to do so (to prevent resizing issues) we could potentially lock the window so a user can't resize the window to be any narrower than this.  Ideally, we would make the absolute minimum width a bit narrower like 800px, but if that doesn't work we can go for something like 1250px or at the very most 1280px
> 
> The 'maximum' minimum window height we could enforce would be something like 680px.  Ideally, a minimum window height would be a bit less than this, e.g. say 600px

On the other hand, this view could be made responsive by simply making items follow the width, as in attached video:

https://user-images.githubusercontent.com/33099791/176868189-aa2963f0-4a81-4d03-9f69-0d6c94f6752e.mp4

Reply:
@osmaczko I'd go for the second proposal, do you have these changes in some brach?